crates/holochain_types/src/web_app/web_app_manifest/web_app_manifest_v1.rs

```diff
@@ -0,0 +1,46 @@
+//! WebApp Manifest format, version 1.
+//!
+//! NB: After stabilization, *do not modify this file*! Create a new version of
+
+//! the spec and leave this one alone to maintain backwards compatibility.
+
+/// Version 1 of the App manifest schema
+#[derive(Clone, Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
+#[serde(rename_all = "snake_case")]
+#[cfg_attr(feature = "arbitrary", derive(arbitrary::Arbitrary))]
+pub struct WebAppManifestV1 {
+    /// Name of the App. This may be used as the installed_app_id.
+    pub name: String,
+
+    /// Web UI used for this app, packaged in a .zip file
+    pub ui: WebUI,
+
+    /// The Cell manifests that make up this app.
+    pub happ_manifest: AppManifestLocation,
+}
+
+/// Web UI .zip file that should be associated with the happ
+#[derive(Clone, Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
+#[serde(rename_all = "snake_case")]
+#[cfg_attr(feature = "arbitrary", derive(arbitrary::Arbitrary))]
+pub struct WebUI {
+    /// Where to find this UI.
```

Comment:
@zippy I've purposefully left out the `name` fields specified in the hackmd here and in `AppManifestLocation` because of YAGNI: I don't understand what they should specify, nor have any use for they in the launcher, and I think it adds to the confusion more than clarifies things.

Reply:
In the end we'll want names to distuguish different UIs, but for now this is ok